Microsoft.Toolkit.Mvvm/Input/Interfaces/IRelayCommand.cs

```diff
@@ -16,5 +16,10 @@ public interface IRelayCommand : ICommand
         /// Notifies that the <see cref="ICommand.CanExecute"/> property has changed.
         /// </summary>
         void NotifyCanExecuteChanged();
+
+        /// <summary>
+        /// Gets a value indicating whether this command can always be executed. True if the command was created without execution status logic.
+        /// </summary>
+        bool CanAlwaysExecute { get; }
```

Comment:
```suggestion
        /// Gets a value indicating whether this command can always be executed. True if the command was created without execution status logic.
        /// </summary>
        bool CanAlwaysExecute { get; }

        /// <summary>
        /// Notifies that the <see cref="ICommand.CanExecute"/> property has changed.
        /// </summary>
        void NotifyCanExecuteChanged();
```

Put property above method? I know StyleCop enforces this for classes, but I guess not for interfaces?

Reply:
Yes, you're absolutely right. Should be like suggested by you if we would merge it. Thanks @michael-hawker